### DISCUSSION TITLE
How to increase read timeout for charts

### DISCUSSION THREAD

**Discussion by Kaliumhexacyanoferrat** [netdata/netdata#12174] [Q&A]
When fetching data for a larger time period on a rather slow system ...

![image](https://user-images.githubusercontent.com/4992119/154683686-033ac823-3504-44ab-91b2-74ca61c1eba5.png)

.. the client issues read timeouts because the server takes too long to fetch the data:

![Capture](https://user-images.githubusercontent.com/4992119/154684200-e00faf1a-870b-4964-9f80-8787ce77ffaf.PNG)

or in the network tab:

![image](https://user-images.githubusercontent.com/4992119/154685763-5147cd5e-c728-4281-9353-0ff13bc70657.png)

Is there a way to set the timeout to a higher value than 15 seconds, e.g. by configuration?

**Comment by ilyam8** [comment]
Hi, @Kaliumhexacyanoferrat.

That looks like a bug and likely you will need to open a report in https://github.com/netdata/dashboard.

Let's ask first @jjtsou @jacekkolasa

**Reply by stelfrag** [reply]
It will be interesting to check how fast your agent will respond directly to such request so maybe checking **localhost:19999/api/v1/data?chart=system.cpu&after=-2592000**      (2592000 being 30 days X 86400 seconds per day)

(probably a good idea to do from the command line)

**Reply by Kaliumhexacyanoferrat** [reply]
Thank you very much for your answers. I have now increased "update very" on the system, which has a very positive effect on the response times. In 28 days I will tackle the problem again 😀
